src/main/java/io/jenkins/plugins/synopsys/security/scan/service/scan/blackduck/BlackDuckParametersService.java

```diff
@@ -178,6 +202,22 @@ private void setDownloadUrl(BlackDuck blackDuck, String value) {
         }
     }
 
+    private void setSearchDepth(BlackDuck blackDuck, Integer value) {
+        if (value != null) {
+            Search search = new Search();
+            search.setDepth(value);
+            blackDuck.setSearch(search);
+        }
+    }
+
+    private void setConfigPath(BlackDuck blackDuck, String value) {
+        if (value != null) {
+            Config config = new Config();
+            config.setPath(value);
+            blackDuck.setConfig(config);
+        }
+    }
+
```

Comment:
Add some unit tests to cover **Lines 136-219**

Reply:
Done